src/NSubstitute/Exceptions/RedundantArgumentMatcherException.cs

```diff
@@ -20,19 +59,13 @@ public RedundantArgumentMatcherException(string message) : base(message)
             IEnumerable<IArgumentSpecification> allSpecifications)
         {
             return
-                $"Not all the argument specifications (e.g. Arg.Is, Arg.Any) were bound during the last call.{Environment.NewLine}" +
-                $"It could mean that previously you passed specification to a method, which is not handled" +
-                $" by the NSubstitute (e.g. method is not virtual, or doesn't belong to a substitute). Another reason" +
-                $" might be that argument specification type doesn't match the actual argument type, but code compiles due" +
-                $" to an implicit cast (e.g. between 'int' and 'double'). In any case this exception is an indication" +
-                $" of the unexpected NSubstitute usage and most likely your test code behaves not as intended.{Environment.NewLine}" +
-                $"Please use the diagnostics information below to detect and fix the defective places.{Environment.NewLine}" +
-                $"{Environment.NewLine}" +
-                $"Remaining (non-bound) argument specifications:{Environment.NewLine}" +
-                $"{string.Join(Environment.NewLine, remainingSpecifications)}{Environment.NewLine}" +
-                $"{Environment.NewLine}" +
-                $"All argument specifications:{Environment.NewLine}" +
-                $"{string.Join(Environment.NewLine, allSpecifications)}";
+                BaseMessage +
+                $"Remaining (non-bound) argument specifications:{ItemSeparator}" +
```

Comment:
Personally, I feel that interpolation would work better here - we are splitting message parts across the whole file 😖

Reply:
See https://github.com/zvirja/NSubstitute/pull/4#discussion_r170766208. I don't feel strongly about it. :)